src/main/resources/templates/about.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.w3.org/1999/xhtml">
+<head>
+    <meta charset="UTF-8">
+    <title>About</title>
+    <link rel="stylesheet" href="../static/css/style.css" th:href="@{/css/style.css}">
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans&display=swap" rel="stylesheet">
+</head>
+<body>
+<header>
+    <div id="leftButtons">
+        <a th:href = "@{/}">Home</a>
+        <a th:href = "@{/about}">About</a>
+    </div>
+    <div id="rightButtons">
+        <a th:href = "@{/login}">Log in</a>
+    </div>
+</header>
+
+<main>
```

Comment:
Everything in these 3 html pages repeats except content inside `<main>...</main>`. Look for [Thymeleaf Layouts](https://www.thymeleaf.org/doc/articles/layouts.html) to reuse this layout, so that making changes happens only in 1 place.